silk-core/src/main/java/net/silkmc/silk/core/mixin/player/MixinServerPlayer.java

```diff
@@ -0,0 +1,22 @@
+package net.silkmc.silk.core.mixin.player;
+
+import net.minecraft.server.level.ServerPlayer;
+import net.minecraft.world.entity.item.ItemEntity;
+import net.minecraft.world.item.ItemStack;
+import net.silkmc.silk.core.event.player.PlayerItemEvents;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
+
+@Mixin(ServerPlayer.class)
+public class MixinServerPlayer {
+
+    @Inject(method = "drop(Lnet/minecraft/world/item/ItemStack;ZZ)Lnet/minecraft/world/entity/item/ItemEntity;", at = @At("HEAD"))
```

Comment:
this might not be the only function for dropping items

Reply:
it is the only function for dropping items from the player